src/core/i18n/en/translation.en.json

```diff
@@ -647,6 +647,7 @@
       "description": "We apologize, but the content you requested is not available. It may have been deleted, or you may not have the permission to access it, such as if it was put in draft mode."
     },
     "createFull": "Add collaboration tool",
+    "createShort": "Collaboration tool",
```

Comment:
I think I'd do `"$t(common.collaborationTool)"` and add `collaborationTool` there in `common`, I've been about to add it already a few times.